core/math/vector2.h

```diff
@@ -89,10 +89,18 @@ struct _NO_DISCARD_ Vector2 {
 		return Vector2(MIN(x, p_vector2.x), MIN(y, p_vector2.y));
 	}
 
+	Vector2 minf(real_t p_scalar) const {
```

Comment:
I would add `_FORCE_INLINE_` for `min*` `max*` methods in headers.  `_FORCE_INLINE_` reduces compile time because the compiler doesn't have to analyse the method to inline it.

Reply:
As per the comment elsewhere I won't add it unless it's warranted, should be done with care, and as it's not done for the existing ones I'll not do it here either, it'll be inlined if it's appropriate anyway